src/main/scala/com/microsoft/partnercatalyst/fortis/spark/dto/FortisEvent.scala

```diff
@@ -23,7 +21,7 @@ case class Analysis(
   locations: List[Location] = List(),
   sentiments: List[Double] = List(),
   moods: List[Tag] = List(),
-  genders: List[Tag] = List(),
+  //genders: List[Tag] = List(),
```

Comment:
No biggie if we want to leave it - just want to confirm this is intentional or if we meant to remove `genders`.